src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala

```diff
@@ -396,7 +396,7 @@ trait EntityPage extends HtmlPage {
 
       {
         if (Set("epfl", "EPFL").contains(tpl.universe.settings.docfooter.value))
-          <div id="footer">Scala programming documentation. Copyright (c) 2003-2016 <a href="http://www.epfl.ch" target="_top">EPFL</a>, with contributions from <a href="http://www.lightbend.com" target="_top">Lightbend</a>.</div>
+          <div id="footer">Scala programming documentation. Copyright (c) 2003-2016 <a href="http://www.epfl.ch" target="_top">EPFL</a>, with contributions from <a href="http://www.lightbend.com" target="_top">Lightbend</a>. Get involved on <a href="https://github.com/scala/scala" target="_top">GitHub</a>.</div>
```

Comment:
Could we deep-link to the github page that corresponds to the documented file, using `templateToPath(tpl)`?